src/vm/excep.cpp

```diff
@@ -5532,6 +5534,26 @@ DefaultCatchHandlerExceptionMessageWorker(Thread* pThread,
         }
 
         PrintToStdErrA("\n");
+
+#if defined(FEATURE_EVENT_TRACE) && !defined(FEATURE_PAL)
+        // Send the log to Windows Event Log
```

Comment:
The check for ShouldLogInEventLog() in the original is missing.

Reply:
The original code gc protected throwable and then checked:

     if (IsException(gc.throwable->GetMethodTable()))

The code to handle the case where that wasn't true is missing. You can hit that case by throwing an object that doesn't derive from System.Exception. Its uncommon and I think not authorable in C#, but CLI does allow it and the code was handling it before so lets preserve it.